augly/video/functional.py

```diff
@@ -171,7 +188,6 @@ def augment_audio(
     assert callable(audio_aug_function), (
         repr(type(audio_aug_function).__name__) + " object is not callable"
     )
-
     func_kwargs = helpers.get_func_kwargs(
```

Comment:
```suggestion

    func_kwargs = helpers.get_func_kwargs(
```

Is there any reason to delete this newline?

Reply:
Nope, my bad it must've accidentally happened while I was trying to manually keep this in sync with what's in `facebookresearch:main`